feg/cloud/go/services/feg_relay/servicers/feg_to_gw_relay.go

```diff
@@ -62,21 +62,20 @@ func getHwIDFromIMSI(ctx context.Context, imsi string) (string, error) {
 	return "", fmt.Errorf("could not find gateway location for IMSI: %s", imsi)
 }
 
-func validateFegContext(ctx context.Context) error {
-	fegId := protos.GetClientGateway(ctx)
-	if fegId == nil {
-		ctxMetadata, _ := metadata.FromIncomingContext(ctx)
-		errorStr := fmt.Sprintf(
-			"Failed to get Identity of calling Federated Gateway from CTX Metadata: %+v",
-			ctxMetadata,
-		)
-		glog.Error(errorStr)
-		return fmt.Errorf(errorStr)
+func getHwIDFromTeid(ctx context.Context, teid string) (string, error) {
+	gw := protos.GetClientGateway(ctx)
+	servedIds, err := getFegServedIds(gw.GetNetworkId())
+	if err != nil {
+		return "", err
 	}
-	if !fegId.Registered() {
-		return fmt.Errorf("federated gateway not registered")
+	for _, nid := range servedIds {
+		hwId, err := directoryd.GetHWIDForSgwCTeid(nid, teid)
```

Comment:
nit: the casing for ID is inconsistent. `hwId` -> `hwID`?

Reply:
i am using the same nomenclature we use in the other functions on this same file (see line 56)

Reply:
Yeah I'm ok with either. Then we should fix the function name to be `getHwIdFromTeid`

Reply:
Although I think in most of our code we use ID over Id. It's not that big of a deal, but it would be good to be consistent wrt function name + variable names

Reply:
ohh and add to that that some of those names are on protos, so changing them is kind of complex since we will create incompatibilities

Reply:
Yeah maybe not worth the effort then :)

Reply:
FWIW and I realize protos make this awkward, google naming style for Go would be ID.

Reply:
changed Id for ID to match the naming